File: LearnGrowListen/ChapterFourGreat/LargestNumber.java
import java.util.Scanner;

public class LargestNumber {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
	System.out.print("Enter a number: ");
	int number = input.nextInt();

		int largest = 0;

 	for(int count = 1;count <= 10;count++){
		if(count == 10){
		    System.out.println(count);
		}

	}

	if(number > largest){
		largest = number;
	}


   }
}

File: LearnGrowListen/ChapterFourGreat/PYTHONTHREE/Largest_number.py
first_Number = (input('Enter the first number:')
second_Number = (input('Enter the first number:')
third_Number = (input('Enter the first number:')

sum = first_Number + second_Number + third_Number
print('Sum:',sum)

average = sum / 3
print('Average:',average)

product = first_Number * second_Number * third_Number
print('Product:',product)

smallest = firstNumber
largest = firstNumber

if second_Number < smallest:
	smallest = second_Number

if third_Number < smallest:
	smallest = third_Number 

	print('Smallest:',smallest)


if second_Number > largest:
	largest = second_Number

if third_Number > largest:
	largest = third_Number 

	print('Largest:',largest)